src/sbvr-api/sbvr-utils.ts

```diff
@@ -1067,9 +1067,12 @@ export const getAffectedIds = Promise.method(
 				}
 				const { idField } = resourceTable;
 
-				_.set(request.odataQuery, ['options', '$select'], {
+				if (request.odataQuery.options == null) {
```

Comment:
This change should be marginally better performance but primarily allows us to make use of the typings to validate the
```ts
request.odataQuery.options.$select = ...
```
assignment and also to avoid the need for a `!` in
```ts
delete request.odataQuery.options.$expand;
````